packages/lib/src/stories/Login.stories.ts

```diff
@@ -15,12 +17,14 @@ const meta: Meta<LoginProps> = {
   render: (args) => html`
     <sesamy-login
       .buttonText=${args.buttonText}
+      .outline=${ifDefined(args.lang)}
       @login=${args.onLogin}
     ></sesamy-login>
```

Comment:
Is storybook using lit, or are we adding the support or it? 🤔

Reply:
Storybook is using lit so it's nothing that is bundled.